test/linalg/blas.jl

```diff
@@ -326,3 +326,108 @@ end
     @test triu(A[1,:] * A[1,:]') ≈ BLAS.her!('U', one(real(elty)), view(A, 1, :), zeros(elty, 5, 5))
     @test tril(A[1,:] * A[1,:]') ≈ BLAS.her!('L', one(real(elty)), view(A, 1, :), zeros(elty, 5, 5))
 end
+
+struct WrappedArray{T,N} <: AbstractArray{T,N}
+    A::Array{T,N}
+end
+
+Base.size(A::WrappedArray) = size(A.A)
+Base.getindex(A::WrappedArray, i::Int) = A.A[i]
+Base.getindex(A::WrappedArray{T, N}, I::Vararg{Int, N}) where {T, N} = A.A[I...]
+Base.setindex!(A::WrappedArray, v, i::Int) = setindex!(A.A, v, i)
+Base.setindex!(A::WrappedArray{T, N}, v, I::Vararg{Int, N}) where {T, N} = setindex!(A.A, v, I...)
+Base.unsafe_convert(::Type{Ptr{T}}, A::WrappedArray{T}) where T = Base.unsafe_convert(Ptr{T}, A.A)
+Base.stride(A::WrappedArray, i::Int) = stride(A.A, i)
```

Comment:
Perhaps before this line you can test that `strides(A::WrappedArray)` and `stride(A::WrappedArray, i)` both throw an error, and that afterward it does not? In a sense, the main contribution of this PR is *removing* a fallback method, so we should test that the fallback doesn't accidentally get reintroduced.

Or you could define `WrappedArray2` and test for the error.

Reply:
I assume by "throws error" you mean `@test_deprecated` since the fallback is still there?

Reply:
Right indeed. :+1: